CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
-## 0.0.1 - 2015-04-09
+## 0.0.1 - 2016-06-15
```

Comment:
Rather than changing the date, add a new line like `## 0.0.2 - 2016-06-15` and below that indicate under an `### Added` section that we increased the Python version to 2.7.11